Makefile

```diff
@@ -8,6 +8,7 @@ PROJECTS_PYCODESTYLE="sdk cli toolchain"
 BUILD_PARAMS=
 LEGION_VERSION=0.11.0
 SANDBOX_PYTHON_TOOLCHAIN_IMAGE=
+LEGIONCTL_NONINTERACTIVE=1
 CREDENTIAL_SECRETS=.secrets.yaml
 ROBOT_FILES=**/*.robot
```

Comment:
What do you think about logout command which will be used to clear credentials in the config file?

Reply:
Commented above - yes, I will add

Reply:
Done